### PR TITLE
0.1.0 docs

### PR DESCRIPTION
**Pull request by yoptar**
- [x] README warning about new data location
- [x] on disc space requirement notes for models that depend on big files
- [x] "how to use" examples at the beginning of page for each model
- [x] warning about changes in API and configs

### CODE REVIEW

**Comment by dilyararimovna** [comment]
I think it is also important to duplicate information about models' memory consumption in `docs/intro/features.rst` in tables with information about pre-trained models.

**Comment by my-master** [comment]
It would be nice to have all external links in `features.rst` outlined in a separate place, e. g. bottom of the page because links repeat and it is more eye-catching.

**Comment by nikolay-bushkov** [comment]
And yeah... let's wait for all models to have their docs updated...

File: docs/components/classifiers.rst [hunk 1]
```diff
@@ -155,7 +158,7 @@ providing corresponding name of the config file (see above):
 
 ::
 
-    python deep.py download configs/classifiers/intents_dstc2.json
+    python deeppavlov/deep.py download configs/classifiers/intents_dstc2.json
```
Comment (yoptar) [inline docs/components/classifiers.rst]:
```suggestion
    python -m deeppavlov download configs/classifiers/intents_dstc2.json
```

File: docs/components/classifiers.rst [hunk 2]
```diff
@@ -164,18 +167,26 @@ etc. The flag ``-d`` provides downloading all the required components.
...
+
+    snips_model = build_model(configs.classifiers.intents_snips , download=True)
+    snips_model(["Hello! What is the weather in Boston tomorrow?"])
 
-    python deep.py interact configs/classifiers/intents_dstc2.json
+or from command line:
+
+.. code:: bash
+
+    python deeppavlov/deep.py interact configs/classifiers/intents_dstc2.json [-d]
```
Comment (yoptar) [inline docs/components/classifiers.rst]:
```suggestion
    python -m deeppavlov interact configs/classifiers/intents_dstc2.json [-d]
```

File: docs/components/classifiers.rst [hunk 3]
```diff
@@ -164,18 +167,26 @@ etc. The flag ``-d`` provides downloading all the required components.
...
 
 or
 
-::
+.. code:: bash
 
-    python deep.py interactbot configs/classifiers/intents_dstc2.json -t <TELEGRAM_TOKEN>
+    python deeppavlov/deep.py interactbot configs/classifiers/intents_dstc2.json -t <TELEGRAM_TOKEN> [-d]
```
Comment (yoptar) [inline docs/components/classifiers.rst]:
```suggestion
    python -m deeppavlov interactbot configs/classifiers/intents_dstc2.json -t <TELEGRAM_TOKEN> [-d]
```

Comment (dilyararimovna) [reply docs/components/classifiers.rst]: I've copied from Mishanya's example=)

File: docs/components/classifiers.rst (hunk 2 above)
Comment (dilyararimovna) [reply docs/components/classifiers.rst]: I've copied from Mishanya's example=)

File: docs/components/classifiers.rst (hunk 1 above)
Comment (dilyararimovna) [reply docs/components/classifiers.rst]: fixed

File: docs/components/classifiers.rst (hunk 2 above)
Comment (dilyararimovna) [reply docs/components/classifiers.rst]: fixed

File: docs/components/classifiers.rst (hunk 3 above)
Comment (dilyararimovna) [reply docs/components/classifiers.rst]: fixed

File: docs/components/ner.rst [hunk 4]
```diff
@@ -404,7 +404,7 @@ Or from command line:
 
 .. code:: bash
 
-    python deeppavlov/deep.py interact deeppavlov/configs/ner/ner_ontonotes.json [-d]
+    python -m deeppavlov interact deeppavlov/configs/ner/ner_ontonotes.json [-d]
```
Comment (dilyararimovna) [inline docs/components/ner.rst]: python -m deeppavlov interact ner_ontonotes [-d]

File: docs/components/tfidf_ranking.rst [hunk 5]
```diff
@@ -58,15 +56,13 @@ Run the following to interact with the **English** ranker:
 
 .. code:: bash
 
-    cd deeppavlov/
-    python deep.py interact configs/doc_retrieval/en_ranker_tfidf_wiki.json -d
+    python -m deeppavlov interact en_ranker_tfidf_wiki.json -d
```
Comment (dilyararimovna) [inline docs/components/tfidf_ranking.rst]: python -m deeppavlov interact en_ranker_tfidf_wiki -d

**Pull request merged**